Buddies/Controls/FilterSearchBar.swift

```diff
@@ -152,12 +275,16 @@ class FilterSearchBar : UISearchBar, UISearchBarDelegate {
         // Store request params #NoRaceConditions
         self.lastSearchParams = myParams
         
+        let start = DateRangeSliderDelegate.instance.getSliderDate(sliderValue: CGFloat(myParams.dateMin))
+        let end = DateRangeSliderDelegate.instance.getSliderDate(sliderValue: CGFloat(myParams.dateMax))
+        let distance = Int(FilterSearchBar.metersPerMile * myParams.maxMilesAway)
+        
         // Load data from algolia!
         api.searchActivities(withText: myParams.filterText,
                              matchingAnyTopicOf: displayDelegate?.getTopics(),
-                             startingAt: myParams.when?.start,
-                             endingAt: myParams.when?.end,
-                             upToDisatnce: myParams.maxMetersAway) {
+                             startingAt: start,
+                             endingAt: end,
+                             upToDisatnce: distance) {
```

Comment:
awesome! at some point we'll have to assign someone to get the actual user's current location... it ignores `upToDistance` if you don't specify coordinates. Not an issue for this PR though